### PR TITLE
🌱 Bump k8s to v1.26.0

### PR DESCRIPTION
**Pull request by kashifest**
This PR also modifies MACAddressPolicy in preKubeAdmCommands for ubuntu tempaltes which should be none on ubuntu 22.04. This change was added by https://github.com/Nordix/metal3-dev-tools/pull/592 but for some this is not getting persisted in the node image. Until that is solved we will exclusively set it here.

### CODE REVIEW

**Comment by kashifest** [comment]
/hold
I will remove test commit once review is done.

**Review by lentzi90 — COMMENTED** [review]
Looks good, let's drop the test commit

**Review by furkatgofurov7 — COMMENTED** [review]
/retitle 🌱 Bump k8s to v1.26.0

File: scripts/ci-e2e.sh [hunk 1]
```diff
@@ -14,8 +14,8 @@ source "${REPO_ROOT}/scripts/environment.sh"
 # Clone dev-env repo
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=main
+M3_DEV_ENV_REPO="https://github.com/Nordix/metal3-dev-env.git"
```
Comment (Rozzii) [inline scripts/ci-e2e.sh:17]: I think you can already revert this 2 changed lines as your branch has been merged.

**Review by Rozzii — CHANGES_REQUESTED** [review]

File: scripts/ci-e2e.sh (hunk 1 above)
Comment (kashifest) [reply scripts/ci-e2e.sh]: Done

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by kashifest** [comment]
/test-ubuntu-integration-main

/hold cancel
Passed e2e tests
https://jenkins.nordix.org/view/Metal3/job/metal3_capm3_main_e2e_test_ubuntu/124/consoleFull
https://jenkins.nordix.org/view/Metal3/job/metal3_capm3_main_e2e_test_centos/108/consoleFull
https://jenkins.nordix.org/view/Metal3/job/metal3_capm3_main_e2e_upgrade_test_ubuntu/168/consoleFull

**Pull request merged**